.src/multi_threading.inl

```diff
@@ -131,6 +131,16 @@ namespace jluna
         other._value = nullptr;
     }
 
+    template<typename T>
+    Task<T>::operator unsafe::Value*()
+    {
+        auto* res = _value->_value;
+        if (res == nullptr)
+            return jl_nothing;
+        else
+            return res;
```

Comment:
```suggestion
        if (_value == nullptr)
            return jl_nothing;
        else
            return _value->_value;
```

Reply:
on move `Task::_value` is set to `nullptr`, which would trigger a segfault on `_value->_value`